backend/app/models.py

```diff
@@ -142,3 +150,13 @@ class Image(db.Model):
 
     def __repr__(self):
         return '<Image %r>' % self.name
+
+
+class RevokedTokenModel(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    jti = db.Column(db.String(120))
```

Comment:
What is `jti`?

Reply:
jti = JWT ID 
this Class stores revoked tokens

Reply:
No need to short a shortcut :) Call it `jwt_id`